PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -204,4 +190,4 @@
 
 muonTask = cms.Task(slimmedMuonsUpdated,isoForMu,ptRatioRelForMu,slimmedMuonsWithUserData,finalMuons,finalLooseMuons )
 muonMCTask = cms.Task(muonsMCMatchForTable,muonMCTable)
-muonTablesTask = cms.Task(muonFSRphotons,muonFSRassociation,muonMVATTH,muonMVALowPt,muonTable,fsrTable)
+muonTablesTask = cms.Task(muonFSRphotons,muonFSRassociation,muonMVATTH,muonMVALowPt,muonTable)
```

Comment:
I just realized we can also skip running muonFSRphotons,muonFSRassociation here (and remove their definition at L90:L100) since they are now replaced by leptonFSRphotons.
It does not make a difference since fsrTable is no longer written here, but they are now run with no effect...

Reply:
indeed, good idea to clean up these not-needed producer already in the PR.